libraries/testkit/admin.py

```diff
@@ -405,6 +406,32 @@ def get_sgreplicate2_active_tasks(self, db, expected_tasks=1):
             time.sleep(1)
         return active_resp_data
 
+    def replication_status_poll(self, db, repl_id, max_times):
```

Comment:
Is there a way to convert max_tries to timeout? It might be easier for future users to get an understanding what the timeout is, rather than them calculating it when the call the function

Reply:
Yes that makes sense, method adjusted